cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -208,12 +208,12 @@ protected boolean removeEldestEntry(Map.Entry<UUID, UserTaskInfo> eldest) {
    * @param parameters Parsed parameters from http request, or null if the parsing result is unavailable.
    * @return The list of {@link OperationFuture} for the linked UserTask.
    */
-  public List<OperationFuture> getOrCreateUserTask(HttpServletRequest httpServletRequest,
-                                                   HttpServletResponse httpServletResponse,
-                                                   Function<String, OperationFuture> function,
-                                                   int step,
-                                                   boolean isAsyncRequest,
-                                                   CruiseControlParameters parameters) {
+  public synchronized List<OperationFuture> getOrCreateUserTask(HttpServletRequest httpServletRequest,
```

Comment:
-- Discussed offline -- 
Currently wondering why:
1. `List<OperationFuture> _futures` is empty?
2. do we use `synchronized (_sessionKeyToUserTaskIdMap) {` inside `synchronized` methods?